libraries/Microsoft.Bot.Builder.AI.QnA/Models/RankerTypes.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace Microsoft.Bot.Builder.AI.QnA
+{
+    /// <summary>
+    /// Enumeration of types of ranking.
+    /// </summary>
+    public enum RankerTypes
```

Comment:
As much as I love enums, they're very fragile from a forward compatibility perspective. 

Please consider making these just string constants. That way we can later add new ones w/o breaking compat.

Reply:
Made it as string const as you suggested. Thanks for the suggestion  :)